browser/brave_stats/brave_stats_updater.cc

```diff
@@ -255,6 +263,10 @@ void BraveStatsUpdater::OnSimpleLoaderComplete(
   // In case the first call was blocked by our timer.
   (void)MaybeDoThresholdPing(0);
 
+  // Notify observers
+  for (auto& obs : observers_)
+    obs.OnStatsPingFired();
```

Comment:
it's not quite clear how wallet metrics are connected to the fact that we did a stats ping

Reply:
I believe this is an experiment to see if we can use P3A to get just as reliable data as the stats ping way. So that we can remove the stats ping thing.

Reply:
Yeah, @bbondy is correct. Basically if someone uses the wallet once, we need a way to update the new P3A values. So we could have either the wallet service have its own timer or tie it up to stats, which already runs every day.

Reply:
If this is a matter of daily timer, I'd suggest we add its own timer to the wallet service. Adding an intercomponent dependencies just because one of them can do a daily method call doesn't look like a good design?

Reply:
switched to a timer.